src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -42,41 +46,77 @@ public static class CreateMutationBuilder
                 return inputs[inputName];
             }
 
-            IEnumerable<InputValueDefinitionNode> inputFields =
-                objectTypeDefinitionNode.Fields
-                .Where(f => FieldAllowedOnCreateInput(f, databaseType, definitions))
+            // The input fields for a create object will be a combination of:
+            // 1. Scalar input fields corresponding to columns which belong to the table.
+            // 2. Complex input fields corresponding to tables having a foreign key relationship with this table.
```

Comment:
For entities that are related only through config file (no FK constraints), the create operation will have fields from related entity.

Please re-write the L51 to reflect that